app/commands/markdown/render_html.rb

```diff
@@ -21,9 +21,19 @@ def link(node)
       out('<a href="', node.url.nil? ? '' : escape_href(node.url), '" target="_blank"')
       out(' title="', escape_html(node.title), '"') if node.title.present?
       out(' rel="nofollow"') if nofollow_links
+      out(link_tooltip_attributes(node))
       out('>', :children, '</a>')
     end
 
+    def link_tooltip_attributes(node)
+      link_match = %r{^(?<url>https://exercism.io)?/tracks/(?<track>[a-zA-Z0-9_-]+)/(?<type>concept|exercise)s/(?<slug>[a-zA-Z0-9_-]+)}.match(node.url) # rubocop:disable Layout/LineLength
```

Comment:
Rather than `[a-zA-Z0-9_-]`, maybe it's safer to use `[^/]`. In case people put unicode in slugs etc (which I believe we don't disallow)

Reply:
(Maybe also add a test with a unicode char in the slug to be sure?)

Reply:
Good suggestion